contrib/devtools/Makefile

```diff
@@ -99,7 +99,7 @@ proto-tools-stamp:
 	@go install github.com/regen-network/cosmos-proto/protoc-gen-gocosmos
 
 	@echo "Installing protoc-gen-grpc-gateway..."
-	@go install https://github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
```

Comment:
I spent an hour trying to figure out what was wrong with the tool

Reply:
:cry: go to know for future changes

Reply:
could you open a pr for this fix? I ran into the same issue. I imagine this could save time for others as well